mypy.ini

```diff
@@ -407,6 +407,17 @@ no_implicit_optional = true
 warn_return_any = true
 warn_unreachable = true
 
+[mypy-homeassistant.components.recorder.statistics]
```

Comment:
WE should type this file.

Reply:
The intention is to add strict checks to `homeassistant.components.recorder.statistics` does this exclude it from strict checks instead?

Reply:
Ah yes, you're right. My bad.